orange/src/main/java/com/hartwig/hmftools/orange/OrangeConfig.java

```diff
@@ -73,6 +73,7 @@ public interface OrangeConfig
     String DOID_JSON = "doid_json";
     String COHORT_MAPPING_TSV = "cohort_mapping_tsv";
     String COHORT_PERCENTILES_TSV = "cohort_percentiles_tsv";
+    String SIGNATURES_ETIOLOGIES_TSV = "signatures_etiologies_tsv";
```

Comment:
In general I believe the correct term is "signature_etiologies". 

There is 1 signature_etiology
There are N signature_etiologies 

:)

(affects naming in multiple places)